web/src/hotkey.js

```diff
@@ -1160,11 +1160,11 @@ export function process_hotkey(e, hotkey) {
                     user_topics_ui.toggle_topic_visibility_policy(recent_msg);
                     return true;
                 }
-                return false;
             }
             if (inbox_ui.is_in_focus()) {
                 return inbox_ui.toggle_topic_visibility_policy();
             }
+            return false;
     }
```

Comment:
This comes into use when adding another case here which is used for channel topic list hotkey addition.